run/hello-broken/main_test.go

```diff
@@ -28,32 +28,34 @@ func TestBrokenErrors(t *testing.T) {
 	req := httptest.NewRequest("GET", "/", payload)
 	rr := httptest.NewRecorder()
 
-	os.Setenv("TARGET", "")
-	brokenHandler(rr, req)
+	os.Setenv("NAME", "")
```

Comment:
I might be missing something, but why is this needed?

Reply:
The sample demonstrates a brittle, low-visibility handling of an unset value, and a sane-defaults high visibility approach. I am testing the cases around both.